shared/js/ui/base/localize.js

```diff
@@ -34,6 +36,18 @@ function site(key, options) {
     return i18next.t(`site:${key}`, options)
 }
 
+/**
+ * A helper for accessing the 'report' namespace with a bit of help from Typescript
+ * @template {keyof reportTranslations} K
+ * @template {`${K}.${string}`} F
+ * @param {F} key
+ * @param {Record<string, any>} [options]
+ */
+function report(key, options) {
+    return i18next.t(`report:${key}`, options)
+}
+
 export const ns = {
     site,
+    report,
 }
```

Comment:
I just add these when I change any translations. it's a low-fi way to do type safety based on the JSON files.